src/scenes/CTAScene.tsx

```diff
@@ -6,26 +6,23 @@ import styled from 'styled-components'
 import { CTAShownState } from 'types'
 import { worldLogic } from 'worldLogic'
 
-const SWrapper = styled.div`
-  display: flex;
-  align-items: center;
-  > div:last-of-type {
-    margin-left: 16px;
-  }
-`
-
 const SModalCTA = styled.div`
   position: absolute;
   background-color: var(--bg);
   border-radius: var(--radius);
   margin-top: 32px;
```

Comment:
@michaelfurutin Seems like this margin is 6px on mockups
```suggestion
  margin-top: 6px;
```

Reply:
fixed